source/test/notebook.test.ts

```diff
@@ -90,4 +72,13 @@ describe('Notebook', () => {
     });
   });
 
+  test('notebookRule has created', () => {
+    const template = initialize();
+    template.resourceCountIs('AWS::IAM::Role', 2);
+  });
+
+  test('notebook policy has created', () => {
```

Comment:
make sure the least permissions introduced for submitting hybird job in notebook!

Reply:
these two permissions have different principals.